app/src/lib/palette.js

```diff
@@ -1,7 +1,14 @@
+/* GLOBAL COLORS */
+export const PURPLE = '#597ce4';
```

Comment:
Not sure if we should name variables by color names, it's better to name them by the purpose :)